desktop/apps/editorial_features/routes.coffee

```diff
@@ -46,6 +47,8 @@ Articles = require '../../collections/articles.coffee'
           headers: 'X-Access-Token': req.user?.get('accessToken') or ''
         )
       ]
+      if @curation.get('sub_articles').length
+        cbs.push( @veniceSubArticles.fetch(data: 'ids[]': @curation.get('sub_articles')) )
```

Comment:
I'm being nit-picky, but calling the array of fetches `cbs` (callbacks) might be misleading. Maybe it would make more sense to rename them to `fetches` or `promises`?